tests/unit/test_zowe_core.py

```diff
@@ -387,43 +304,32 @@ def test_secure_values_loading_exception(self, get_pass_func):
 
 class TestValidateConfigJsonClass(unittest.TestCase):
     """Testing the validate_config_json function"""
+    
+    def test_validate_config_json_valid(self):
+        """Test validate_config_json with valid config.json matching schema.json"""
+        path_to_config = FIXTURES_PATH + "/zowe.config.json"
+        path_to_schema = FIXTURES_PATH + "/zowe.schema.json"
 
-    def test_validate_config_json_parameterized(self):
-        """Test validate_config_json with different values"""
-        test_values = [
-            ((matching_config_json, matching_schema_json), True),
-            ((not_matching_config_json, not_matching_schema_json), False),
-        ]
-
-        current_dir = os.getcwd()
-
-        for test_case in test_values:
-            if test_case[1]:
-                with open("config.json", "w") as file:
-                    commentjson.dump(matching_config_json, file)
-
-                with open("schema.json", "w") as file:
-                    commentjson.dump(matching_schema_json, file)
-
-                expected = validate(test_case[0][0], test_case[0][1])
-                result = validate_config_json(current_dir + "/config.json", current_dir + "/schema.json")
+        config_json = json.load(open(path_to_config))
+        schema_json = json.load(open(path_to_schema))
 
-                self.assertEqual(result, expected)
+        expected = validate(config_json, schema_json)
+        result = validate_config_json(path_to_config, path_to_schema)
 
-            else:
-                with open("config.json", "w") as file:
-                    commentjson.dump(not_matching_config_json, file)
+        self.assertEqual(result, expected)
 
-                with open("schema.json", "w") as file:
-                    commentjson.dump(not_matching_schema_json, file)
+    def test_validate_config_json_invalid(self):
+        """Test validate_config_json with invalid config.json that does not match schema.json"""
+        path_to_invalid_config = FIXTURES_PATH + "/invalid.zowe.config.json"
+        path_to_invalid_schema = FIXTURES_PATH + "/invalid.zowe.schema.json"
 
-                with self.assertRaises(Exception) as expected_info:
-                    validate(test_case[0][0], test_case[0][1])
+        invalid_config_json = json.load(open(path_to_invalid_config))
+        invalid_schema_json = json.load(open(path_to_invalid_schema))
 
-                with self.assertRaises(Exception) as actual_info:
-                    validate_config_json(current_dir + "/config.json", current_dir + "/schema.json")
+        with self.assertRaises(Exception) as expected_info:
```

Comment:
Shouldn't you expect here `ValidationError`?

Reply:
Yes, I'll fix that.